### DISCUSSION TITLE
different environment variable for a subset of the replicas in ECS

### DISCUSSION THREAD

**Discussion by faisalil** [aws/copilot-cli#5782] [Q&A]
Hello,

one service I have on ECS is called "workers", currently, all those workers listen to the same queue of tasks that we have. I have 3 replicas.
I want to do a change and make 2 of the machines listen to 1 queue and the other machine listen to a new queue.

What I need to to spin up 2 replicas with a different environment variables than the other machine. I want to avoid creating new service since there is just a lot of env variables and configs that I don't want to change.

**Comment by huanjani** [comment]
Hi! 
See my response here: https://github.com/aws/copilot-cli/issues/5780

**Reply by faisalil** [reply]
this doesn't answer my question: my question, can I have a different enviroment variables for a subset of the machines?
